Metallicity_Stack_Commons/analysis/error_prop.py

```diff
@@ -41,6 +42,18 @@ def write_npz(path: str, npz_files: list, dict_list: list,
     log_verbose(log, "finished.", verbose=verbose)
 
 
+def replace_oiii4363(flux_file: str, detection: np.ndarray,
+                     flux_dict: dict,
+                     log: Logger = log_stdout()):
+
+    flux_limit = OIII4363_flux_limit(flux_file, log=log)
+
+    non_det = np.where(detection == 0.5)[0]
```

Comment:
In Zcal, I refer to the robust limits as 'rlimit' instead of 'non-det'. For consistency, should they be the same?

Reply:
That's an interesting point. There are some `rlimit` reference in the codebase and `rlimit` should be the better term since `detection == 0` is also non-detection. I'll switch this over.